```
_____________ ERROR collecting bodhi/tests/server/test___init__.py _____________
/usr/lib/python2.7/site-packages/_pytest/python.py:468: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib/python2.7/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
E     File "/builddir/build/BUILD/bodhi-3.8.0/bodhi/tests/server/test___init__.py", line 48
E       @mock.patch('bodhi.server.log.exception')
E       ^
E   SyntaxError: invalid syntax
----- generated xml file: /builddir/build/BUILD/bodhi-3.8.0/nosetests.xml ------
```

I guess I've missed ) - yes, amended.

--- a/bodhi/tests/server/test___init__.py
+++ b/bodhi/tests/server/test___init__.py
@@ -43,7 +43,7 @@ class TestExceptionFilter(unittest.TestCase):
 
         self.assertIs(response, request_response)
         exception.assert_called_once_with(
-            "Unhandled exception raised:  OSError('Your money is gone.',)")
+            "Unhandled exception raised:  {}".format(repr(request_response)))
 
     @mock.patch('bodhi.server.log.exception')
     def test_no_exception(self, exception):

--- a/bodhi/tests/server/test_mail.py
+++ b/bodhi/tests/server/test_mail.py
@@ -287,7 +287,8 @@ class Test_SendMail(unittest.TestCase):
         SMTP.assert_called_once_with('smtp.fp.o')
         smtp.sendmail.assert_called_once_with('archer@spies.com', ['lana@spies.com'], 'hi')
         warn.assert_called_once_with(
-            '"recipient refused" for \'lana@spies.com\', SMTPRecipientsRefused(\'nooope!\',)')
+            '"recipient refused" for \'lana@spies.com\', {}'.format(
+                repr(smtp.sendmail.side_effect)))
         smtp.quit.assert_called_once_with()
 
     @mock.patch.dict('bodhi.server.mail.config', {'smtp_server': ''})

